docs/connectors/twitch.md

```diff
@@ -0,0 +1,78 @@
+# Twitch
+
+A connector for [Twitch](https://twitch.tv/).
+
+## Requirements
+
+- A Twitch Account
+- A Twitch App obtained from [Twitch developers page](https://dev.twitch.tv/console/apps)
+- The `code` obtained from the first step of OAuth
+
+## Configuration
+
+```yaml
+connectors:
+  twitch:
+    # required
+    code: "hfu923hfks02nd2821jfislf"  # Code obtained from the first oauth step
+    client-id: "e0asdj48jfkspod0284"
+    client-secret: "kdksd0458j93847j"
```

Comment:
I would maybe change the example values to something like `MYTWITCHCODE`, `MYCLIENTID`, `MYCLIENTSECRET`, etc.